WebRoot/css/zm.css

```diff
@@ -7279,4 +7273,78 @@ input[type="radio"] + label {
 
 .ZmWhiteBlackViewNumberUsed {
 	@WhiteBlackListViewItemsNumberUsed@
-}
\ No newline at end of file
+}
+
+/* Preference signature interface panel */
+.ZmPrefSignaturePage {
+	@PrefSignaturePage@
+}
+
+.ZmPrefSignaturePage hr {
+	@PrefSignaturePageSeparator@
+}
+
+.ZmSignatureNameWrapper {
+	@PrefSignatureNameWrapper@
+}
+
+.ZmSignatureNameWrapper > label {
+	@PrefSignatureNameLabel@
+}
+
+.ZmSignatureNameWrapper .DwtInputField input[type='text'] {
+	@PrefSignatureNameInput@
+}
+
+.ZmSignatureFormatWrapper {
+	@PrefSignatureFormatWrapper@
+}
+
+.ZmSignatureListWrapper {
+	@PrefSignatureListWrapper@
+}
+
+.ZmSignatureListView {
+	@PrefSignatureListView@
+}
+
+.ZmSignatureListView [class*='Row'] TD {
+	@PrefSignatureListRow@
+}
+
+.ZmSignatureListView [class*='Row-selected'] {
+	@PrefSignatureListRowActive@
```

Comment:
I was thinking of having a name PrefSignatureListRowSelected, but this is also ok.